docroot/modules/custom/layout_builder_custom/layout_builder_custom.module

```diff
@@ -840,6 +841,18 @@ function _layout_builder_custom_process_element(array $element, FormStateInterfa
 
     case 'inline_block:uiowa_button':
       $element['field_uiowa_button_link']['widget'][0]['title']['#maxlength'] = 25;
```

Comment:
I think this line needs to be updated to account for multiple buttons (deltas 1 and 2 in widget) to ensure all buttons have a max length.

Reply:
Tested the update, and working great. Thanks @bspeare!